### DISCUSSION TITLE
How to add zooming feature?

### DISCUSSION THREAD

**Discussion by arashi-dev** [idiotWu/smooth-scrollbar#380] [Q&A] [answered]
Hi. it is about 2 days I'm searching for a library like this "smooth-scrollbar" but also have a feature for zooming. finally, after 2 days and nights, I found nothing.
I tried to recreate my own scrollbar or forking this library, but it turned out that I don't have enough knowledge to do it (even after 2 years of learning)

Now, my friends. it's honor to me to know what you think about "how can I do this".
I want to be able to zoom-in and zoom-out while using this library

**Comment by sadeghbarati** [comment, marked as answer]
https://gist.github.com/cirmaciu/40baacc6ac1cb949f9de3732a7d0810e

**Reply by arashi-dev** [reply]
did the job for me. thank you very much

**Reply by dolphin-wood** [reply]
This should be add to the [widely-used plugins list](https://github.com/idiotWu/smooth-scrollbar/projects/6#column-15100590) in the next major release 😆. Thanks!
